README.md

```diff
@@ -124,32 +129,37 @@ This type system has practical significance. For example, when computing pairwis
-|  Mahalanobis         |  `mahalanobis(x, y, Q)`    | `sqrt((x - y)' * Q * (x - y))` |
-|  SqMahalanobis       |  `sqmahalanobis(x, y, Q)`  | ` (x - y)' * Q * (x - y)`  |
-|  WeightedEuclidean   |  `weuclidean(x, y, w)`     | `sqrt(sum((x - y).^2 .* w))`  |
-|  WeightedSqEuclidean |  `wsqeuclidean(x, y, w)`   | `sum((x - y).^2 .* w)`  |
-|  WeightedCityblock   |  `wcityblock(x, y, w)`     | `sum(abs(x - y) .* w)`  |
-|  WeightedMinkowski   |  `wminkowski(x, y, w, p)`  | `sum(abs(x - y).^p .* w) ^ (1/p)` |
-|  WeightedHamming     |  `whamming(x, y, w)`       | `sum((x .!= y) .* w)`  |
+| type name            |  convenient syntax           | math definition     |
+| -------------------- | ---------------------------- | --------------------|
+|  Euclidean           |  `euclidean(x, y)`           | `sqrt(sum((x - y) .^ 2))` |
+|  SqEuclidean         |  `sqeuclidean(x, y)`         | `sum((x - y).^2)` |
+|  Cityblock           |  `cityblock(x, y)`           | `sum(abs(x - y))` |
+|  Chebyshev           |  `chebyshev(x, y)`           | `max(abs(x - y))` |
+|  Minkowski           |  `minkowski(x, y, p)`        | `sum(abs(x - y).^p) ^ (1/p)` |
+|  Hamming             |  `hamming(x, y)`             | `sum(x .!= y)` |
+|  Rogers-Tanimoto     |  `rogerstanimoto(x, y)`      | `2(sum(x&!y) + sum(!x&y)) / (2(sum(x&!y) + sum(!x&y)) + sum(x&y) + sum(!x&!y))` |
+|  Jaccard             |  `jaccard(x, y)`             | `1 - sum(min(x, y)) / sum(max(x, y))` |
+|  CosineDist          |  `cosine_dist(x, y)`         | `1 - dot(x, y) / (norm(x) * norm(y))` |
+|  CorrDist            |  `corr_dist(x, y)`           | `cosine_dist(x - mean(x), y - mean(y))` |
+|  ChiSqDist           |  `chisq_dist(x, y)`          | `sum((x - y).^2 / (x + y))` |
+|  KLDivergence        |  `kl_divergence(x, y)`       | `sum(p .* log(p ./ q))` |
+|  GenKLDivergence     |  `gkl_divergence(x, y)`      | `sum(p .* log(p ./ q) - p + q)` |
+|  RenyiDivergence     |  `renyi_divergence(x, y, k)` | `log(sum(x .* (x ./ y) .^ (k - 1))) / (k - 1)` |
+|  JSDivergence        |  `js_divergence(x, y)`       | `KL(x, m) / 2 + KL(y, m) / 2 with m = (x + y) / 2` |
+|  SpanNormDist        |  `spannorm_dist(x, y)`       | `max(x - y) - min(x - y)` |
+|  BhattacharyyaDist   |  `bhattacharyya(x, y)`       | `-log(sum(sqrt(x .* y) / sqrt(sum(x) * sum(y)))` |
+|  HellingerDist       |  `hellinger(x, y)`           | `sqrt(1 - sum(sqrt(x .* y) / sqrt(sum(x) * sum(y))))` |
+|  Mahalanobis         |  `mahalanobis(x, y, Q)`      | `sqrt((x - y)' * Q * (x - y))` |
+|  SqMahalanobis       |  `sqmahalanobis(x, y, Q)`    | `(x - y)' * Q * (x - y)`  |
+|  MeanAbsDeviation    |  `meanad(x, y)`              | `mean(abs.(x - y))` |
+|  MeanSqDeviation     |  `msd(x, y)`                 | `mean(abs2.(x - y))` |
+|  RMSDeviation        |  `rmsd(x, y)`                | `sqrt(msd(x, y))` |
+|  NormRMSDeviation    |  `nrmsd(x, y)`               | `rmsd(x, y) / (maximum(x) - minimum(x))` |
```

Comment:
Any reason to normalize by the range rather than by the mean? Both forms seem to exist, but I figure the mean will be more robust. Or maybe have it as a separate `CVRMSDeviation` metric since that's how [Wikipedia](https://en.wikipedia.org/wiki/Root-mean-square_deviation#Normalized_root-mean-square_deviation) calls it.